internal/services/chat_history_recap/chat_history_recap.go

```diff
@@ -97,7 +97,7 @@ func (s *ChatHistoryRecapService) SendChatHistoriesRecap() {
 			continue
 		}
 
-		s.Logger.Info("sending chat histories recap for chat %d", chatID)
+		s.Logger.Infof("sending chat histories recap for chat %d", chatID)
```

Comment:
There is a warning here, which I fixed by the way.

Reply:
Thanks!